api/src/main/java/org/openmrs/module/reporting/common/SqlRunner.java

```diff
@@ -35,7 +35,7 @@
  */
 public class SqlRunner {
 
-	private static Log log = LogFactory.getLog(SqlRunner.class);
+    private static final Logger log = LoggerFactory.getLogger(SqlRunner.class);
```

Comment:
I guess I'm ok with this, but this is something that would ideally need to be changed across the entire reporting module codebase for consistency, ideally in a different PR that is focused on this.  If you need this for some particular reason to be changed here, that's OK, but would be good to know what that is.

Reply:
Can we try to change it back, and see what the issue is?